pom.xml

```diff
@@ -297,6 +300,38 @@
                 </plugins>
             </build>
         </profile>
+
+        <profile>
+            <id>jacoco</id>
+            <modules>
+                <!-- Relying on Maven to build this module as the last one. -->
+                <module>coverage-report</module>
```

Comment:
That's an option I tried (with the `windup-bootstrap` module, which comes last right now). Adding a dependency affects Maven compilation classpath, which I pass to the Ant task to generate the report. JaCoCo and Windup have conflicting ASM dependency, so generating the report didn't work. I could specify the entire JaCoCo classpath manually (it's like 5 JARs or so, 4 from JaCoCo and ASM), but I found that more error-prone than this approach. Module order in Maven is already defined by declaration order (among other things).